controllers/k8ssandra/seeds.go

```diff
@@ -127,6 +133,10 @@ func newEndpoints(dc *cassdcapi.CassandraDatacenter, seeds []corev1.Pod) *corev1
 		})
 	}
 
+	for _, seed := range additionalSeeds {
+		addresses = append(addresses, corev1.EndpointAddress{IP: seed})
```

Comment:
Do we want to have these userdefined seeds also as endpoints or added to the additionalSeeds in the CassandraDatacenter?

Reply:
What would be the reason or advantage for doing so?

Reply:
It would separate the user input with automated processing. It would quickly show if some weird seed is because of user's values (when debugging) or because of something k8ssandra-operator pushed. Semantics only.

Reply:
The user's values are already stored in the K8ssandraCluster spec, so I'm not sure I see the benefit of putting them in the CassandraDatacenter spec for debugging. More importantly though, if we set CasssndraDatacenter.Spec.AdditionalSeeds, then we will have both cass-operator and k8ssandra-operator reconciling the Endpoints. I fear that might cause problems.

Reply:
Passing the additional seeds to the cassdc would indeed duplicate the effort (although it makes debugging easier indeed...).
It should be either/or IMO, and the way John implemented it is relevant with how k8ssandra-operator approaches seed management.